multiqc/utils/search_patterns.yaml

```diff
@@ -67,6 +67,8 @@ preseq:
     contents: 'EXPECTED_DISTINCT'
     # contents: 'TOTAL_READS	EXPECTED_DISTINCT'
     # contents: 'TOTAL_BASES	EXPECTED_DISTINCT'
+prokka:
+    contents: 'organism:'
```

Comment:
Prokka doesn't have any standard for what these files are called, so I guess this is as close as we can come unless you find something better someday. At least the prokka parser I wrote makes a few additional checks to make sure, which is better than nothing...